### DISCUSSION TITLE
Exercise 2: Part 1.

### DISCUSSION THREAD

**Discussion by guille21alaman** [sebastian-hofstaetter/teaching#77] [Advanced IR: Exercise Q&A] [answered]
Hi, 

I have a question regarding task 1 of exercise 2. If I understood correctly, what we have to do in this task is to create a new file similar to **fira-22-baseline-qrels.tsv** using the data from the judgments (and optionally from documents and queries tsv files). That is, for each combination of queryId and documentId, assign a label (relevance-level) to it.**  This file should **be named fira-2022.tuples.tsv**  My first question is: is this correct?

Assuming that I understood the task correctly, I proceed with my next questions:

- Do we have to be **creative** and come up with a custom solution for this? Or should we rather stick with methods already explored in current research or **discussed in the lectures**?
- Could you e**laborate a bit more on the evaluation part** (i.e. meta-judgment and quality of the aggregations) ?
- Should we expect better results from our custom solution than from the provided baselines?

Thank you very much. Hope the questions are fine :)

**Comment by sophiaalthammer** [comment, marked as answer]
Hi,
yes you understood the assignment, that is correct.

To your first question: you can be creative, you dont have to, as long as you can argue why you take a certain approach it is fine.
I am not sure what your question is regarding meta-judgements and quality of aggregations, here we introduced several methods in the evaluation lecture 02 how the quality of judgements can be measured and analyzed.
No, you dont have to expect better results from your custom solution.

I hope that answers your questions!

**Reply by guille21alaman** [reply]
Yes. It is clear now. Thank you :)
